intake_bluesky/jsonl.py

```diff
@@ -58,12 +96,13 @@ def _load(self):
                 self._filename_to_mtime[filename] = mtime
                 with open(filename, 'r') as file:
                     try:
-                        name, run_start_doc = json.loads(file.readline())
+                        name, start_doc = json.loads(file.readline())
                     except json.JSONDecodeError:
                         if not file.readline():
                             # Empty file, maybe being written to currently
                             continue
```

Comment:
While we're here I noticed a small flaw that would come up if we ever encountered a malformed JSON file. If we hit a `json.JSONDecodeError` and the file is _not_ empty but in fact corrupted, we should re-raise that error rather than proceeding to `upsert`, where we will fail with a more confusing error. Can you add a line here, below and outside of the `if` block, to `raise`?